packages/lit-dev-content/site/tutorials/content/wc-to-lit/13.md

```diff
@@ -0,0 +1,49 @@
+## Congratulations!
+
+You've built a "vanilla" web component and converted it to a Lit-based web component!
+
+You've also converted an unreadable <litdev-code-language-display>
+  <span slot="js">101</span>
+  <span slot="ts">99</span>
+</litdev-code-language-display>-line vanilla web component with a 27-line html template to a <litdev-code-language-display>
+  <span slot="js">75</span>
+  <span slot="ts">68</span>
+</litdev-code-language-display>-line Lit-based web component template-and-all!
```

Comment:
the space before the `<litdev-code-language-display>` gets stripped so it looks like
> You've also converted an unreadable101-line ...
and
> 27-line html template to a75-line ...

It appears it only does that in the deployed preview, but locally it preserves the space. Maybe there's some kind of linting that's stripping the space happening in CI/CD?

I wonder if replacing the space with `&#32;` would make sure it's preserved? (found the code here https://stackoverflow.com/questions/38661784/breaking-space-opposite-of-non-breaking-space)